usaspending_api/common/management/commands/matview_runner.py

```diff
@@ -75,21 +79,37 @@ def generate_matview_sql(self):
         self.matview_dir.mkdir()
 
         # IF using this for operations, DO NOT LEAVE hardcoded `python3` in the command
-        exec_str = "python3 {} --quiet --dest={}/ --batch_indexes=3".format(MATVIEW_GENERATOR_FILE, self.matview_dir)
+        # Create main list of Matview SQL files
+        exec_str = f"python3 {MATVIEW_GENERATOR_FILE} --quiet --dest={self.matview_dir}/ --batch_indexes=3"
         subprocess.call(exec_str, shell=True)
 
+        # Create SQL files for Chunked Universal Transaction Matviews
+        for matview, config in self.chunked_matviews.items():
+            exec_str = f"python3 {MATVIEW_GENERATOR_FILE} --quiet  --dest={self.matview_dir} --file {config['json_filepath']} --chunk-count {self.chunk_count}"
+            subprocess.call(exec_str, shell=True)
+
     def cleanup(self):
         """Cleanup files after run"""
         recursive_delete(self.matview_dir)
 
     def create_views(self):
         loop = asyncio.new_event_loop()
         tasks = []
+
+        # Create Matviews
         for matview, config in self.matviews.items():
-            logger.info("Creating Future for {}".format(matview))
+            logger.info(f"Creating Future for matview {matview}")
             sql = (self.matview_dir / config["sql_filename"]).read_text()
             tasks.append(asyncio.ensure_future(async_run_creates(sql, wrapper=Timer(matview)), loop=loop))
 
+        # Create Chunked Matviews
+        for matview, config in self.chunked_matviews.items():
+            for current_chunk in range(0, self.chunk_count):
```

Comment:
Including the default starting index of 0 isn't typical, unless if a different step size is also provided.

```suggestion
            for current_chunk in range(self.chunk_count):
```